worker/src/lib.rs

```diff
@@ -78,6 +79,11 @@ impl From<(String, EdgeRequest)> for Request {
 }
 
 impl Request {
+    pub fn new(url: &str) -> Request {
+        let req = EdgeRequest::new_with_str(url).unwrap();
+        Request::from((String::from("fetch"), req))
```

Comment:
You can use `"fetch".to_string()` to get a `String` instance from a literal.

Reply:
done